code/headers/st7735_buffered.hpp

```diff
@@ -0,0 +1,67 @@
+#pragma once
+
+#include <hwlib.hpp>
+#include <type_traits>
+
+#include "st7735.hpp"
```

Comment:
Moet dit niet met `<>`?

Reply:
ik vind zelf dat als de include in de huidige directory zit dat hij met de ```""``` moet zodra hij doormiddel van de search gevonden moet worden vind ik eigenlijk dat de ```<>``` gebruikt moet worden maarja